src/math/numbers.c

```diff
@@ -176,5 +176,7 @@ uint32_t crc32(uint32_t base, const void *data, uint32_t bytes)
 	return ~crc;
 }
 
+#if !CONFIG_SOC_MIMX9352_A55
```

Comment:
CONFIG_SOC_MIMX9352_A55 != 64 bits

Reply:
@andyross can you help here?

Reply:
This is a libgcc symbol implementing a long divide (exactly which operation I'd have to look up, there are lots of variants).  The bug here is presumably that some platforms (presumably only the NXP listed) are aarch64 devices with have a hardware divide, and so libgcc.a on those platforms doesn't include it.  So trying to EXPORT_SYMBOL() ends up hitting a link failure trying to extract an address (because it doesn't exist).  This seems not unreasonable.

I think if one wanted to nitpick:

1. It should probably check something more generic, like CONFIG_64BIT, CONFIG_SOC_FAMILY_ARM64 (also X86_64 if we ever want to do 64 bit fuzzing!), or a GCC arch symbol like __x86_64__ or __ARM_ARCH_8A.
2. It's not a SOF symbol to begin with , and this EXPORT_SYMBOL() belongs upstream in Zephyr llext somewhere.